### DISCUSSION TITLE
is WG_HOST for the UI, the vpn, or both?

### DISCUSSION THREAD

**Discussion by phirestalker** [wg-easy/wg-easy#722] [Q&A] [answered]
I would like to only allow systems on my LAN to access the configuration page (or webui). I would like to be able to access the VPN from the internet. I currently have WireGuard set up through traefik working well.
Is this possible? Do I use the internal domain name for WG_HOST to keep the UI on the LAN?
I did not do a search because I am unsure of the terms to use.

**Comment by eandersons** [comment, marked as answer]
`WG_HOST` is

> The public hostname of your VPN server

and it is used in peers (clients) configuration as the endpoint address.
So, answering your question, `WG_HOST` is used for VPN connection.

Access to the web user interface can be controlled with proxy server (it can be done with Traefik's [`IpWhitelist` middleware](https://doc.traefik.io/traefik/middlewares/tcp/ipwhitelist/#ipwhitelist)) and/or firewall.

**Reply by phirestalker** [reply]
Just to be clear, I should use the same hostname to access the VPN and the UI? I had forgotten about whitelisting. If I have to use that then it will provide the same protection.

**Reply by eandersons** [reply]
I am not completely sure, but I think that the user interface can have any domain name as it is not coupled with the VPN endpoint.
